src/bindings/html/view.js

```diff
@@ -3,22 +3,16 @@
 import { documentReady, getDirection } from './shims';
 import { getResourceLinks, getMeta } from './head';
 import {
-  setAttributes, getAttributes, translateFragment, translateMutations,
+  initMutationObserver, translateRoots, observeRoot, disconnectRoot
+} from './observer';
+import {
+  setAttributes, getAttributes, translateFragment, translateMutations
 } from './dom';
 
-const observerConfig = {
-  attributes: true,
-  characterData: false,
-  childList: true,
-  subtree: true,
-  attributeFilter: ['data-l10n-id', 'data-l10n-args']
-};
-
-const readiness = new WeakMap();
+const priv = new WeakMap();
```

Comment:
Can we get a better name here? `priv` feels in the same universe as `data`, `meta`, and `thing`. :)